_security-analytics/threat-intelligence/api/monitor.md

```diff
@@ -0,0 +1,324 @@
+---
+layout: default
+title: Monitor API
+parent: Threat intelligence APIs
+grand_parent: Threat intelligence
+nav_order: 35
+---
+
+# Monitor API
+
+The threat intelligence Monitor API creates, searches, and updates [monitors](https://opensearch.org/docs/latest/observing-your-data/alerting/monitors/) using the monitor's ID.
+
+
+---
+## Create or update a threat intelligence monitor
+
+Creates or updates a new threat intelligence monitor.
+
+### Path and HTTP methods
+
+The `POST` method creates a new monitor. The `PUT` method updates the monitor.
+
+```json
+POST _plugins/_security_analytics/threat_intel/monitors
+PUT _plugins/_security_analytics/threat_intel/monitors/<monitor_id>
+```
+
+### Request fields
```

Comment:
I'm still seeing an issue with the table formatting:
<img width="776" alt="image" src="https://github.com/user-attachments/assets/92a3f06b-8d09-495f-b05d-bcbaad951262">

Reply:
The table is formatting fine for me. Let me take a screenshot.